module/VuFind/src/VuFind/View/Helper/Root/SearchBox.php

```diff
@@ -478,9 +489,20 @@ protected function getCombinedHandlers($activeSearchClass, $activeHandler, array
             $label = $settings['label'][$i];
 
             if ($type == 'VuFind') {
-                $options = $this->getOptionsForTarget($target);
                 $j = 0;
-                $basic = $options->getBasicHandlers();
+                $basic = null;
```

Comment:
Since you're continuing out of the catch clause, I don't think you strictly need to initialize `$basic` prior to the try.